### ISSUE TITLE
Delete `.(` warning in React Stateless Component

### ISSUE THREAD

**Issue by tapas4java** [prettier/eslint-plugin-prettier#33]
Hi,

Thanks for this plugin. Prettier is keep complaining about `(` and `)` around JSX on a stateless component. Is it a bug or are there any rule to disable this warning?
```
const App = ({ session }) => (
   <div>Hello!</div>
);
```

**Comment by tapas4java** [comment]
Thanks @not-an-aardvark for the confirmation. I will raise a ticket on `prettier` repo.
